asserts/assertstest/assertstest.go

```diff
@@ -299,11 +326,28 @@ func NewStoreStack(authorityID string, rootPrivKey, storePrivKey asserts.Private
 		panic(err)
 	}
 
+	err = db.ImportKey(genericPrivKey)
+	if err != nil {
+		panic(err)
+	}
+	genericKey := NewAccountKey(rootSigning, genericAcct, map[string]interface{}{
+		"name":  "serials",
+		"since": ts,
+	}, genericPrivKey.PublicKey(), "")
```

Comment:
This sort of logic looks nicer if we pull out the map into a variable created before the call.